tools/autograd/load_derivatives.py

```diff
@@ -315,6 +315,11 @@ def saved_variables(formula, args):
             'suffix': '_dim',
             'type': 'int64_t',
         }),
+        # replace self.is_conj() with self_conjugate
+        (r'{}.is_conj\(\)', {
```

Comment:
Is this always a valid thing to do? Is there any case where this flag can be changed inplace on a Tensor?

Reply:
Yes it's always valid. Even if the flag is changed inplace, the old saved value is the correct one to be used for backwards, similar to how you can resize a tensor that was saved for backwards, but you'll get the old size in the backwards itself (and `resize_` doesn't actually do a VC update)